docs/README.md

```diff
@@ -37,13 +38,10 @@ List of known adopters:
 | Vultr | KubeAI can be deployed on Vultr Managed Kubernetes using the application marketplace. | [Vultr](https://www.vultr.com) |
 | Arcee | Arcee uses KubeAI for multi-region, multi-tenant SLM inference. | [Arcee](https://www.arcee.ai/) |
 
-If you are using KubeAI and would like to be listed as an adopter, please make a PR.
+If you are using KubeAI and would like to be listed as an adopter, please [submit a PR](https://github.com/substratusai/kubeai/edit/main/docs/README.md).
 
 ## Local Quickstart
 
-
-<video controls src="https://github.com/user-attachments/assets/711d1279-6af9-4c6c-a052-e59e7730b757" width="800"></video>
```

Comment:
We should have a video demo imo. So people can quickly see what they would get after installing.

Reply:
I am hoping to get rid of OpenWebUI, which is showcased in the video

Reply:
We can keep the video showcasing an example chatUI (e.g. OpenWebUI) even though KubeAI doesn't include installation of it. I prefer to keep the video until we have a new video though.